x-pack/platform/plugins/shared/security/server/authorization/register_privileges_with_cluster.ts

```diff
@@ -85,10 +89,22 @@ export async function registerPrivilegesWithCluster(
 
     await clusterClient.asInternalUser.security.putPrivileges({ body: expectedPrivileges });
     logger.debug(`Updated Kibana Privileges with Elasticsearch for ${application}`);
+
+    securityTelemetry.recordPrivilegeRegistrationDuration(performance.now() - startTime, {
```

Comment:
Just double checking that we do not want to log a duration if privileges are equal (return on ln 71). Do we care about capturing the time to get/compare the privileges if we're not changing privileges?